SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SchemaGenObjectStub.h

```diff
@@ -17,6 +17,18 @@ class USchemaGenObjectStub : public UObject
 	bool BoolValue;
 };
 
+UCLASS()
+class USchemaGenObjectStubCondOwnerOnly : public UObject
+{
+	GENERATED_BODY()
+public:
```

Comment:
```suggestion
public:
	virtual void GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const override;

```

Reply:
not sure what's the difference

Reply:
Like other classes defined in the SchemaGenObjectStub.h file. All of them didn't have GetLifetimeReplicatedProps declaration.